conan/tools/sbom/cycloneDX.py

```diff
@@ -0,0 +1,73 @@
+
+def cyclonedx_1_4(graph, **kwargs):
```

Comment:
Need to get also the different interfaces for other cases like: 
- Conan graph.json manifest
- Conan lockfile
To align in something common